rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java

```diff
@@ -139,8 +139,9 @@ private CachingRlsLbClient(Builder builder) {
             timeProvider);
     RlsRequestFactory requestFactory = new RlsRequestFactory(lbPolicyConfig.getRouteLookupConfig());
     rlsPicker = new RlsPicker(requestFactory);
-    ManagedChannelBuilder<?> rlsChannelBuilder =
-        helper.createResolvingOobChannelBuilder(rlsConfig.getLookupService());
+    ManagedChannelBuilder<?> rlsChannelBuilder = helper.createResolvingOobChannelBuilder(
+        rlsConfig.getLookupService(), helper.getUnsafeChannelCredentials());
+    rlsChannelBuilder.overrideAuthority(helper.getAuthority());
```

Comment:
Has overrideAuthority been enhanced to make this safe?

Reply:
Just send out #7854